app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
```

Comment:
@xprazak2 ,

Sorry but I am disagree with your explanation that label-{tag} determines the row, but result-filter-tag decides the column. 
When you say $('.label-{tag}.result-filter-tag'), it will give you DOM element which having both these CSS classes i.e `label-{tag}` and `result-filter-tag`.

From shared example, it will give you `span` i.e `<span class="label label-info result-filter-tag">value to filter</span>`

$('.result-filter-tag') will give you all DOM elements which having only `result-filter-tag` CSS class. 
That's why it should work as in your example also it will return `span` element i.e `<span class="label label-info result-filter-tag">value to filter</span>`